gatherpress.php

```diff
@@ -5,7 +5,7 @@
  * Description:       Powering Communities with WordPress.
  * Author:            The GatherPress Community
  * Author URI:        https://gatherpress.org/
- * Version:           0.32.2
+ * Version:           0.32.3
```

Comment:
Don't worry about versions. I handle that in released. You're also branched off main not develop. You need to be branched off and merged into develop.